src/ripple/app/paths/RippleCalc.cpp

```diff
@@ -149,7 +149,12 @@ RippleCalc::Output RippleCalc::rippleCalculate (
         {
             JLOG (j.error()) << "Exception from flow: " << e.what ();
             if (!useFlowV1Output)
-                Rethrow();
+            {
+                // return a tec so the tx is stored
+                path::RippleCalc::Output exceptResult;
+                exceptResult.setResult(tecINTERNAL);
+                return exceptResult;
+            }
```

Comment:
Is it kosher to continue with `flowV1SB.apply (view);` and returning `flowV1Out` when `useFlowV1Output` is true and an exception is thrown? I assume it is okay although the code as written looks worrisome.